src/vmspawn/vmspawn.c

```diff
@@ -60,6 +60,8 @@ static int arg_tpm = -1;
 static char *arg_linux = NULL;
 static char *arg_initrd = NULL;
 static bool arg_qemu_gui = false;
+static bool arg_network_tap = false;
+static bool arg_network_user_mode = false;
```

Comment:
even if these are separate switches, i'd still encode this internally via your enum, i.e. replace these two lines by `static QemuNetworkStack arg_network_stack = QEMU_NET_NONE;` and then just make the switches switch between the values.